src/serialized-analysis.ts

```diff
@@ -44,7 +44,10 @@ export interface SourceLocation {
 }
 
 export interface AnalyzedPackage {
```

Comment:
Actually... I kind of hate "metadata" in a name usually. It's often pretty content free, but then again so is "descriptor", "entity",  "feature", etc. We don't have a plethora of great choices here...